README.md

```diff
@@ -86,3 +86,34 @@ frontend/model/challenges.py
 frontend/routes/api/challenges.py
 frontend/api.yaml # This is a specification of the challenges API (file above) using OpenAPI.
 ```
+### Sprint 3
+During our third sprint, we put all the components of our project together and got them functioning. Our challenge team
+made further progress on their challenge sets. We had enough cipher challenges to convert it ot a challenge set of its
+own and make the remaining challenges a second challenge set which we named web-based as all the challenges there are
+targeted towards teaching the user how a directory traversal or similar web-based attack works.
+
+For the UI we managed to implement a 3rd party library that converts markdown into HTML for being able to display
+documents that are more easily written to the screen as an HTML object. Additionally, we implemented a special useFetch
```

Comment:
> For the UI we managed to implement a 3rd party library that converts markdown into HTML for being able to display documents that are more easily written to the screen as an HTML object.

I'd say something more along the lines of "For the UI, we utilized the third-party react-markdown library to render Markdown text as HTML, which we can then insert into the page and style it accordingly".